internal/cephfs/controllerserver.go

```diff
@@ -228,7 +228,7 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 
 	defer func() {
 		if err != nil {
-			if !errors.Is(err, ErrCloneInProgress) {
+			if !isCloneRetryError(err) {
```

Comment:
Need to use isCloneRetryError in line 244 below as well. The gRPC error return would not be set correct if we only check for ErrCloneInProgress at the specified line.

Reply:
Yes correct, updated error check now